up-l1/mqtt.adoc

```diff
@@ -35,8 +35,85 @@ MQTT is an OASIS standard messaging protocol for the Internet of Things (IoT). I
 
 For more information, please refer to https://mqtt.org/
 
-
 == Specifications
 
+=== UAttribute Mapping
+
+MQTT 5 supports custom header fields, and we leverage this to map certain UAttribute values into the MQTT header.
+
+[cols="1,1"]
+|===
+| MQTT Header Property Code | Value (key:value)
+| UserProperty
+| ("upriority","{UAttribute.priority}")
+| CorrelationData
+| ("reqId", "{UAttribute.reqId}")
+| UserProperty
+| ("sink","{UAttribute.sink}") # Should this be serialized to MicroUri?
```

Comment:
I dunno enough about MQTT5, but from what I can see of the spec in [my previous comment over here](https://github.com/eclipse-uprotocol/up-spec/pull/75/files#r1522025611), probably?